app/views/links/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Submit Link</h1>
+
+<% if @link.errors.any? %>
+  <ul>
+    <% @link.errors.uniq.each do |error_field| %>
+      <li><%= "#{ error_field.to_s.humanize } " + @link.errors[error_field].join('; ') %></li>
+    <% end %>
+  </ul>
+<% end %>
```

Comment:
seems to be a slightly different format:
<%= render partial: "form", locals: {zone: @zone} %>